marlowe-runtime/src/Language/Marlowe/Runtime/Transaction/BuildConstraints.hs

```diff
@@ -35,7 +84,137 @@ buildApplyInputsConstraints
                    -- in the contract.
   -> Redeemer v -- ^ The inputs to apply to the contract.
   -> Either ApplyInputsError (TxConstraints v)
-buildApplyInputsConstraints = error "not implemented"
```

Comment:
@jhbertra should I make this function effectful so I can grab the current time if it is necessary to define lower time range boundry?

Reply:
Yes, that sounds like a good idea.